App/Components/Avatar.tsx

```diff
@@ -74,14 +74,16 @@ class Avatar extends React.Component<Props, State> {
     if (isLocalUser && nodeStarted && localAvatar && widthNumber) {
       // Render TextileImage
       return (
-        <TextileImage
-          style={{ ...(this.props.style || {}), width, height, borderRadius: this.state.borderRadius }}
-          target={localAvatar}
-          index={0}
-          forMinWidth={widthNumber}
-          resizeMode={'cover'}
-          onLayout={this.onImageLayout}
-        />
+        <View style={{ width, height, borderRadius: this.state.borderRadius, overflow: 'hidden' }}>
+          <TextileImage
+            style={{ ...(this.props.style || {}), width, height, borderRadius: this.state.borderRadius }}
+            target={localAvatar}
+            index={0}
+            forMinWidth={widthNumber}
+            resizeMode={'cover'}
+            onLayout={this.onImageLayout}
+          />
+        </View>
```

Comment:
Did you test if the same border radius problem exists in the other cases of this `if` statement? I'm guessing it does... When I was looking into this yesterday, the problem seemed to be that the `onLayout={this.onImageLayout}` (which results in the border radius being set) wasn't being triggered on Android. The same technique is used in those other `if` cases.

Reply:
I didn't explicitly... but good point. Though I did see a URI based avatar and a blank Icon one (before URI resolved) from my iOS beta install, and they were both looked fine. We could wrap the whole if else block in a `View` just to be sure. Re: layout event... the border radius is coming through so it must be fired at some point. I logged it in render at one point... was zero for a couple renders, then good values.

Reply:
Cool, yea there's a good chance that when i saw it not firing, it was just because of all the problems with AS and emulators and devices. I really can't explain how broken everything Android seemed yesterday. It was very confusing.

Reply:
K, cool. You ok w/ not doing the full View wrap unless we see that issue? I pretty sure the other cases aren't an issue... probably why we hadn't seen this before all over the place.

Reply:
yea sounds good to me